### PR TITLE
touchdetector: drop MVDTool with latest version.

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

File: var/spack/repos/builtin/packages/touchdetector/package.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class Touchdetector(CMakePackage):
     git      = "git@bbpgitlab.epfl.ch:hpc/touchdetector.git"
 
     version('develop', submodules=True)
+    version('5.6.0.remote', branch='remote-filtering', submodules=True)
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/touchdetector/package.py]: not as per `semantic versioning` scheme :) 

Is this temporary for testing or for the deployment? If for deployment, commit id would be better than branch.

Comment (matz-e) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: ah damn. yeah, testing :)

Comment (matz-e) [reply var/spack/repos/builtin/packages/touchdetector/package.py]:
```suggestion
```

**Review by sergiorg-hpc — DISMISSED** [review]

File: var/spack/repos/builtin/packages/touchdetector/package.py [hunk 2]
```diff
@@ -56,6 +54,10 @@ class Touchdetector(CMakePackage):
     depends_on('libxml2', when='@:4.4')
     depends_on('zlib', when='@:4.4')
 
+    depends_on('morphio@2.0.8:', when='@4.5:5.1')
+    depends_on('mvdtool@2.1.0:', when='@5.1.1:5.6.0')
```
Comment (sergiorg-hpc) [inline var/spack/repos/builtin/packages/touchdetector/package.py]: I had the review approved, but then I noticed the `5.6.0` here and in line 47. Shouldn't it be `5.5.9999`?

Comment (sergiorg-hpc) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: (Sorry for approving and dismissing the review, just trying to be careful here)

Comment (matz-e) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: You're right. Keep forgetting about in/exclusive ranges

Comment (matz-e) [inline var/spack/repos/builtin/packages/touchdetector/package.py]:
```suggestion
    depends_on('mvdtool@2.1.0:', when='@5.1.1:5.5.999')
```

**Review by sergiorg-hpc — APPROVED** [review]

File: var/spack/repos/builtin/packages/touchdetector/package.py (hunk 2 above)
Comment (sergiorg-hpc) [reply var/spack/repos/builtin/packages/touchdetector/package.py]: Yeah... It happens :). Ty for taking care of it, approved!

**Pull request merged**